cookbook/email/spool.rst

```diff
@@ -55,10 +55,15 @@ swiftmailer with the memory option, use the following configuration:
             'spool' => array('type' => 'memory')
         ));
 
-Spool Using a File
+Spool Using Files
```

Comment:
I'm sorry, I don't get it. The Table of contents is updated automatically. Why should we do this?